<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>163</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="8" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/scale_z.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="8">
    <widget class="QSpinBox" name="zScaleSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSlider" name="zScaleSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QCheckBox" name="labelsCheckBox">
     <property name="text">
      <string>Labels</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="nodesCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Nodes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>143</width>
        <height>177</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="scrollLayout"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>resfresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="edgesCheckBox">
     <property name="text">
      <string>Edges</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QCheckBox" name="volumesCheckBox">
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="ceilsCheckBox">
     <property name="text">
      <string>Ceils</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QCheckBox" name="wallsCheckBox">
     <property name="text">
      <string>Wall</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QCheckBox" name="sectionsCheckBox">
     <property name="text">
      <string>Sections</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>zScaleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zScaleSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zScaleSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zScaleSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
